app/views/replies/_reply.html.haml

```diff
@@ -9,7 +9,7 @@
         %span.comment-time-block
           replied to
           = link_to reply.comment.name, "#comment#{reply.comment.id}", title: "View comment"
-          = link_to "#reply#{reply.id}", title: "View reply" do
+          = link_to "#{application_path(reply.comment.application)}\#reply#{reply.id}", title: "View reply" do
```

Comment:
This definitely does the job, but the implementation should be consistent with how the links on comments work, see https://github.com/openaustralia/planningalerts/blob/master/app/views/comments/_comment_meta_sentence.html.haml#L9

Basically that's using a helper method that gets this logic out of the view https://github.com/openaustralia/planningalerts/blob/master/spec/helpers/comments_helper_spec.rb#L23-L30